apps/tangle-website/src/libs/notion/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from './notion';
+export * from './types';
+export * from './server-constants';
```

Comment:
The only Notion files / functions required will be for fetching privacy / terms pages. Everything else can be removed.